exercises/space-age/canonical-data.json

```
@@ -66,13 +120,31 @@
       "expected": 0.46
     },
     {
-      "description": "age on Neptune",
+      "description": "2 Earth years in uranus years",
+      "property": "age",
+      "input": {
+        "planet": "Uranus",
+        "seconds": 63115200
+      },
+      "expected": 0.02
+    },
+    {
+      "description": "57.7 Earth years in Neptune years",
       "property": "age",
       "input": {
         "planet": "Neptune",
         "seconds": 1821023456
       },
       "expected": 0.35
-    }
+    },
+    {
+      "description": "2 Earth years in Neptune years",
+      "property": "age",
+      "input": {
+        "planet": "Neptune",
+        "seconds": 63115200
+      },
+      "expected": 0.01
+    },
```

Comma after last array item is invalid.

```suggestion
    }
```

---

I've never seen this, "commit suggestion" thing. Neat!